test/Makefile.include

```diff
@@ -0,0 +1,176 @@
+    test/functional/listsinceblock.py \
+    test/functional/listtransactions.py \
+    test/functional/maxblocksinflight.py \
+    test/functional/maxuploadtarget.py \
+    test/functional/mempool_limit.py \
+    test/functional/mempool_packages.py \
+    test/functional/mempool_reorg.py \
+    test/functional/mempool_resurrect_test.py \
+    test/functional/mempool_spendcoinbase.py \
+    test/functional/merkle_blocks.py \
+    test/functional/multi_rpc.py \
+    test/functional/net.py \
+    test/functional/nodehandling.py \
+    test/functional/nulldummy.py \
+    test/functional/p2p-acceptblock.py \
+    test/functional/p2p-compactblocks.py \
+    test/functional/p2p-feefilter.py \
+    test/functional/p2p-fullblocktest.py \
+    test/functional/p2p-leaktests.py \
+    test/functional/p2p-mempool.py \
+    test/functional/p2p-segwit.py \
+    test/functional/p2p-timeouts.py \
+    test/functional/p2p-versionbits-warning.py \
+    test/functional/preciousblock.py \
+    test/functional/prioritise_transaction.py \
+    test/functional/proxy_test.py \
+    test/functional/pruning.py \
+    test/functional/rawtransactions.py \
+    test/functional/README.md \
+    test/functional/receivedby.py \
+    test/functional/reindex.py \
+    test/functional/replace-by-fee.py \
+    test/functional/rest.py \
+    test/functional/rpcbind_test.py \
+    test/functional/rpcnamedargs.py \
+    test/functional/segwit.py \
+    test/functional/sendheaders.py \
+    test/functional/signmessages.py \
+    test/functional/signrawtransactions.py \
+    test/functional/smartfees.py \
+    test/functional/test_framework \
+    test/functional/test_runner.py \
+    test/functional/txn_clone.py \
+    test/functional/txn_doublespend.py \
+    test/functional/wallet-accounts.py \
+    test/functional/walletbackup.py \
+    test/functional/wallet-dump.py \
+    test/functional/wallet-hd.py \
+    test/functional/wallet.py \
+    test/functional/zapwallettxes.py \
+    test/functional/zmq_test.py \
+    test/functional/test_framework/__init__.py \
+    test/functional/test_framework/address.py \
+    test/functional/test_framework/authproxy.py \
+    test/functional/test_framework/bignum.py \
+    test/functional/test_framework/blockstore.py \
+    test/functional/test_framework/blocktools.py \
+    test/functional/test_framework/comptool.py \
+    test/functional/test_framework/coverage.py \
+    test/functional/test_framework/key.py \
+    test/functional/test_framework/mininode.py \
+    test/functional/test_framework/netutil.py \
+    test/functional/test_framework/script.py \
+    test/functional/test_framework/siphash.py \
+    test/functional/test_framework/socks5.py \
+    test/functional/test_framework/test_framework.py \
+    test/functional/test_framework/util.py
+
+FUNCTIONAL_TESTS = \
+    nodehandling \
+    httpbasics \
+    invalidblockrequest \
+    blockchain \
+    rpcnamedargs
+
+.PHONY: all_functional_tests functional_tests util_tests
```

Comment:
why these targets need to be here in the new file instead of alongside unit_tests and check-local?
It seems nice for EXTRA_DIST and FUNCTIONAL_TESTS, but that's it IMO.